build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1961,9 +1961,30 @@ message Compressor {
 
     // Brotli compression.
     BROTLI = 3;
+
+    // Zstandard compression with dictionary.
+    //
+    // When this is used, the server MUST advertise the dictionaries by
+    // including
+    // [ZstdDictionaryRegistry][build.bazel.remote.execution.v2.ZstdDictionaryRegistry]
+    // digest in CacheCapabilities.
+    ZSTD_DICT = 4;
   }
 }
 
+message ZstdDictionaryRegistry {
```

Comment:
Any specific reason we have "Registry" part of the message & digest field name? I'd just call this "ZstdDictionaries" or "ZstandardDictionaries".

Reply:
I was unsure about using plural forms, so I sidestepped with `registry` which has been relatively popular in the naming convention recently.

I don't mind going back to ZstdDictionaries.